doc/release/upcoming_changes/27088.change.rst

```diff
@@ -0,0 +1,3 @@
+PR `27088 <https://github.com/numpy/numpy/pull/27088>`_  contains changes to
+bump the musllinux image to 1_2 from 1_1. This is because the 1_1 image is
```

Comment:
No need to mention the PR, towncrier does that for us. Also, the user-facing change is the wheel build, not so much the CI run image.
```suggestion
Bump the musllinux CI image and wheels to 1_2 from 1_1. This is because 1_1 is
```